tiledb/libtiledb.pyx

```diff
@@ -3825,7 +3825,11 @@ cdef class Array(object):
                     _raise_ctx_err(ctx_ptr, rc)
                 if is_empty:
                     return None
-                results.append((start_buf.item(0), end_buf.item(0)))
+
+                if start_size > 0 and end_size > 0:
+                    results.append((start_buf.item(0), end_buf.item(0)))
+                else:
+                    results.append(('', ''))
```

Comment:
I think this should be `(None,None)` to match other coe paths. cc @nguyenv for decision though.

Reply:
Yes I agree to make it `(None, None)`